From 59edcbffa7f860c5700769e043c513fc728b49d3 Mon Sep 17 00:00:00 2001
From: Tom Chauvel <tom.chauvel@proton.me>
Date: Fri, 13 Sep 2024 14:07:41 +0200
Subject: [PATCH] test

---
 rmtfs.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rmtfs.service.in b/rmtfs.service.in
index abd12df..f070828 100644
--- a/rmtfs.service.in
+++ b/rmtfs.service.in
@@ -2,7 +2,7 @@
 Description=Qualcomm remotefs service
 
 [Service]
-ExecStart=RMTFS_PATH/rmtfs -r -P -s
+ExecStart=RMTFS_PATH/rmtfs -r -s -o /var/lib/rmtfs
 Restart=always
 RestartSec=1
 
-- 
2.45.0.windows.1
